src/pages/Search.vue

```diff
@@ -51,6 +51,16 @@ export default class Search extends Vue {
   /** The selected property type filter */
   propertyTypeFilter = '';
 
+  /** The current sorted field (column) */
+  sortedField = 'GHGIntensity';
+
+  /** The direction of the sorted field (column) */
+  sortedDirection = 'desc';
```

Comment:
If this can only be `'asc'` or 'desc'`, we can type it as such, which adds type safety, by doing:

```ts
sortedDirection: 'asc' | 'desc' = 'desc';
```

That way someone later can't accidentally do `sorteDirection = 'ascending'`.